aptos-move/framework/aptos-framework/sources/configs/staking_config.move

```diff
@@ -190,4 +194,24 @@ module aptos_framework::staking_config {
     public entry fun test_update_rewards_invalid_denominator_should_fail(aptos_framework: signer) acquires StakingConfig {
         update_rewards_rate(&aptos_framework, 1, 0);
     }
+
+    public fun initialize_for_test(
```

Comment:
what's the difference here compared to the public fun initialize except for naming? do you mean to have test_only for it?

Reply:
It's just naming to clearly convey this is only for tests/testnets.